src/zarr/core/group.py

```diff
@@ -503,6 +506,8 @@ async def open(
             to load consolidated metadata from a non-default key.
         """
         store_path = await make_store_path(store)
+        if not store_path.store.supports_consolidated_metadata:
```

Comment:
```suggestion
        if not store.supports_consolidated_metadata:
```

Reply:
I don't think this works, store could be a string for example.